packages/tsconfig.json

```diff
@@ -26,6 +26,7 @@
         "pretty": true,
         "removeComments": true,
         "sourceMap": true,
-        "downlevelIteration": true
+        "downlevelIteration": true,
+        "preserveSymlinks": true
```

Comment:
this solves:

https://github.com/microsoft/TypeScript/issues/42873
`make build-react-ui-components-standalone`

```
src/Tabs/tabs.tsx:106:12 - error TS2742: The inferred type of 'handleTabNavItemClick' cannot be named without a reference to '.pnpm/@types+react@16.9.34/node_modules/csstype'. This is likely not portable. A type annotation is necessary.

106     public handleTabNavItemClick = (id: string | number) => {
               ~~~~~~~~~~~~~~~~~~~~~
```